internal/olmbindata/olm_versions.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package olmbindata
+
+func GetAvailableOLMVersions() map[string]struct{} {
```

Comment:
The funcs which are exported should always have a comment. I know that it is in the internal.  However, it is helpful to maintainers and if one day we decide to move the impl to a pkg the godoc will be gen with it

```suggestion
// GetAvailableOLMVersions will return all bindata versions ...
func GetAvailableOLMVersions() map[string]struct{} {
```